src/main/java/juja/microservices/gamification/slackbot/dao/RestGamificationRepository.java

```diff
@@ -0,0 +1,33 @@
+package juja.microservices.gamification.slackbot.dao;
+
+import juja.microservices.gamification.slackbot.model.DailyAchievement;
+import org.springframework.http.*;
+import org.springframework.web.client.RestTemplate;
+
+import javax.inject.Inject;
+
+/**
+ * @author Danil Kuznetsov
+ */
+
+
+public class RestGamificationRepository implements GamificationRepository {
+
+    private final RestTemplate restTemplate;
+
+    private final String URL_SEND_DAILY = "/achieve/daily";
+
+    @Inject
```

Comment:
1. I use @inject, because  it's  part Java CDI, and i haven't relation to Spring framework
2. i use method "constructor injection" because  define all fields in constructor it's pure and plain java solution

Thanks for review :)